src/vivarium_gates_iv_iron/constants/results.py

```diff
@@ -58,11 +59,22 @@
 NON_COUNT_TEMPLATES = []
 
 POP_STATES = ("living", "dead", "tracked", "untracked")
-SEXES = ("male", "female")
-# TODO - add literals for years in the model
-YEARS = ()
+SEXES = ("female")
```

Comment:
you will need this to read `("female",)` - i.e. with a trailing comma - in order for this to be a tuple rather than just a string.